lookout/style/format/features.py

```diff
@@ -43,6 +43,7 @@ def __init__(self,  value: str, start: Position, end: Position,
         self.value = value
         assert start.line >= 1 and start.col >= 1, "start line and column are 1-based like UASTs"
         assert end.line >= 1 and end.col >= 1, "end line and column are 1-based like UASTs"
+        assert y in EMPTY_CLS or start.offset < end.offset, "illegal empty node"
```

Comment:
Could you clarify this line? Without context, I can't understand why it's required

Reply:
It's a test that empty nodes are only of the classes from which we expect empty nodes (-space, -tab and noop)